# Gene set enrichment analysis with multivariate logistic regression

Gene set enrichment analysis (GSEA) summarizes gene-level statistics at the level of predefined gene sets, such as metabolic/signaling pathways. GSEA has two common strategies. The first uses the gene-level statistics to select a gene subset and applies hypergeometric-like tests for the over-representation of each gene set in this subset. The second strategy uses the gene-level statistics to rank all genes and applies Kolmogorovâ€“Smirnov or similar tests for the skewness of a gene set in the overall ranking. Neither of these strategies considers possible confounding variables that bring systematic bias into gene-level statistics.  For example, longer genes generally obtained more sequencing reads from RNA-seq experiments, which gives them the advantage of higher statistical sensitivity to be ranked higher. 

We propose a more sophisticated strategy of GSEA using logistic regression models. In each model, the outcome variable is whether a gene is a member of a predefined gene set and the predictive variables include one or multiple gene-level statistics, such as differential gene expression, as well as zero to any number of confounding variables. Any of the predictive variables could be binomial, categorical, or continuous. Each model will report the accuracy of using the gene-level statistics, while controlling for the confounders, to predict whether a gene is a member of one gene set. This method will have broader applications in genomic studies, especially those involving integrative analysis of multiple data sets.

Requirement: familiar with regression modeling and statistical software, preferably R
